### ISSUE TITLE
Checksum error in additional data loading.

### ISSUE THREAD

**Issue by ooyamams** [ossc-db/pg_bulkload#24]
Checksum error raised when pg_bulkload loads to an existing table if the table size becomes more than 1GB.

Reproduce.
PostgreSQL 9.4.3, pg_bulkload 3.1.8
```
initdb with "--data-checksums".
$ pgbench -i -s 100
$ psql -c "COPY (SELECT * FROM pgbench_accounts WHERE aid <= 7000000) TO '/tmp/test1.csv' WITH CSV;"
$ psql -c "COPY (SELECT * FROM pgbench_accounts WHERE aid > 7000000) TO '/tmp/test2.csv' WITH CSV;"
$ psql -c "CREATE TABLE testtbl (LIKE pgbench_accounts);"
$ cat >/tmp/test1.ctl
OUTPUT = testtbl
INPUT = /tmp/test1.csv
TYPE = CSV
^d
$ pg_bulkload /tmp/test1.ctl
$ psql -c "ALTER TABLE testtbl ADD CONSTRAINT testtbl_pkey PRIMARY KEY (aid);"
$ psql -c "ALTER TABLE testtbl DROP CONSTRAINT testtbl_pkey;"
$ cat >/tmp/test2.ctl
OUTPUT = testtbl
INPUT = /tmp/test2.csv
TYPE = CSV
^d
$ pg_bulkload /tmp/test2.ctl
$ psql -c "ALTER TABLE testtbl ADD CONSTRAINT testtbl_pkey PRIMARY KEY (aid);"

WARNING:  page verification failed, calculated checksum 48076 but expected 0
ERROR:  invalid page in block 130568 of relation base/12146/16431
```

**Comment by amitlan** [comment]
Thanks for reporting the issue.

Could you please tell the write mode used in the control file?  DIRECT or anything else?

**Comment by ooyamams** [comment]
Thanks for the response.

As described at test1.ctl and test2.ctl, we used DIRECT(default) mode.

**Comment by amitlan** [comment]
I see, thanks.

**Comment by amitlan** [comment]
I created a PR.  Please check if it fixes the issue for you.

**Comment by bwtakacy** [comment]
Oh, it's my bug...

**Comment by amitlan** [comment]
No worries, can you please review the changes?

**Comment by bwtakacy** [comment]
Sure:)

Will check.

**Comment by amitlan** [comment]
Oops, for some reason the PR got merged.

**Comment by bwtakacy** [comment]
Yeah, it's ok.

I will re-open this issue if i find some points to be changed.
Thanks for your quick work!

**Comment by amitlan** [comment]
What happened is I pushed some other fix in the master branch without realizing that this bug fix was already committed to the master branch in my local repository.  I forgot to back it out from the master branch after creating a separate branch for PR.

**Comment by bwtakacy** [comment]
Hmm.

There something wrong in commit history in master branch.
The commit in merged PR is not shown in GitHub Web site.
Could you check and fix?

**Comment by amitlan** [comment]
Sorry, some negligence on my part.

I backed out the commit for this bug-fix from the master branch of my local repository and force-pushed to GitHub.  So now there is a merged PR (#25) which is now invalid (because the commit it merged is no longer in the commit history of the master branch).  I have created a new PR after rebasing the bug-fix branch.  You can review that.

Sorry about the inconvenience.

**Comment by bwtakacy** [comment]
Never mind:)

I will check the new one at this night.
Thanks!
